geoapps/utils/utils.py

```diff
@@ -508,7 +508,7 @@ class LineDataDerivatives:
         padding.
     :param residual: Use the residual between the values and the running mean to compute derivatives.
     :param sampling: Sampling interval length (m) used in the FFT. Defaults to the mean data separation.
-    :param smoothing: Number of neighbours used by the :obj:`geoapps.utils.running_mean` routine.
+    :param smoothing: Number of neighbours used by the :obj:`geoapps.test_utils.running_mean` routine.
```

Comment:
Was this intended?

Reply:
Nope, I did a refactor->rename in pycharm and it borked a couple of names.  Just did a find and replace and pushed.  Will see if the tests pass now.